### ISSUE TITLE
Not show tweaks

### ISSUE THREAD

**Issue by Kozekus** [mineek/Serotonin#93]
No tweaks are displayed in the system settings,iPhone 12 iOS 16.2 serotonin 1.2.1

**Comment by Alot1z** [comment]
It seems you have to install a Tweak Settings tweak like https://t.co/flNOE8LucC

**Comment by ShatteredDisk** [comment]
This is correct, but not the right solution. You need to enable tweak injection for the Settings app through Bootstrap.
